eng/performance/scenarios.proj

```diff
@@ -44,14 +44,6 @@
     </Scenario>
   </ItemGroup>
 
-  <!-- Startup FDD publish -->
-  <ItemGroup>
-    <HelixWorkItem Include="@(Scenario -> 'Startup - %(Identity) - FDD Publish')">
-      <PreCommands>$(Python) pre.py publish -f $(_Framework) -c Release</PreCommands>
-      <Command>$(Python) test.py startup --scenario-name &quot;%(Identity)&quot;</Command>
-    </HelixWorkItem>
-  </ItemGroup>
-
```

Comment:
I would comment this out instead of delete it, as we will want to turn it back on at some point.